main.c

```diff
@@ -50,13 +50,9 @@ int main(void) {
         types.stage = select_stage();
         switch(types.stage) {
             case STAGE1:
```

Comment:
Thanks.  This is called `fall through` technique to omit `break`. However, this is difficult to understand that whether `break` is explicitly omitted or not.  So, could you add the following comment above this line?

e.g.

```
// Fall through for selecting stages 1 to 5
```